tests/test_messages.py

```diff
@@ -303,6 +303,37 @@ def test_serializing_decimal():
     assert json.dumps([decimal.Decimal(2.000001)], cls=_DecimalEncoder) == "[2.0]"
 
 
+def test_serializing_custom_types():
+    """
+    validate_payload() raises an exception receives an invalid OCPP message.
+    This exception contains the Call causing the problem.
+
+    The exception is turned into a CallError which in serialized to JSON.
+    https://github.com/mobilityhouse/ocpp/issues/395 tracks a bug where serialization
+    would fails because Call is not serializable.
+
+    This test verifies that fix for that bug. Serializ
```

Comment:
Is there a missing sentence at 'Serializ' or is this a leftover?

Reply:
Good catch. Fixed.